pkg/controllermanager/controller/shoot/shoot_conditions_control.go

```diff
@@ -111,7 +115,8 @@ func (r *shootConditionsReconciler) Reconcile(ctx context.Context, request recon
 	// Update the shoot conditions if needed
 	if gardencorev1beta1helper.ConditionsNeedUpdate(shoot.Status.Conditions, conditions) {
 		r.logger.Debugf("Updating shoot %s conditions", kutil.ObjectName(shoot))
-		if err := r.updateShootConditions(ctx, shoot, conditions); err != nil {
+		shoot.Status.Conditions = conditions
+		if err := r.gardenClient.Status().Update(ctx, shoot); err != nil {
```

Comment:
Does it make sense to add a comment here? Using `Update()` instead of `Patch()` is done very consciously here, so probably we should prevent that this gets reverted somewhen in the future when we are no longer aware of the details.

Reply:
done